### PR TITLE
weather-app Anna Almestål

### PR DESCRIPTION
**Pull request by anna-ip**
This week started off pretty good compare to last week, I met up a couple of class mates and we helped each other out on tue. But on wed at the lesson I realised, I sort of misunderstod the no. 4 and hadn't done the "for each weather" in the forecast, so I did the code along but have then had problems to add the description to my code, so I've sort of feel lika I have used Gaffer tape to put my project together this week 😂.

### CODE REVIEW

**Comment by anna-ip** [comment]
https://weather-app-anna-almestal.netlify.com/

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700&display=swap" rel="stylesheet" />
+  <title>Weather</title>
+</head>
+
+<body>
+  <section class="todays-weather">
+    <div id="container2"></div>
```
Comment (ericaglimsholt) [inline index.html]: `container2` isn't such a good name

File: style.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
+body {
+    display: flex;
+    flex-direction: column;
+    margin: 60px;
+    background-color: #f4f7f8;
+    /*background-image: linear-gradient(#f4f7f8,  #9e9e9e);*/
```
Comment (ericaglimsholt) [inline style.css]: Oh, why didn't you go through with this?

File: style.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    height: 25%;
+  }
+
+  .greet {
+    font-size: 40px;
+    margin: 30px 0;
+  }
+
+  h1 {
+    margin-block-start: 0.50em;
```
Comment (ericaglimsholt) [inline style.css]: Careful here. `margin-block-start` isn't supported on alla browser, so it's better to use `margin-top` for this one https://caniuse.com/#search=margin-block-start

File: style.css [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+p {
+    font-size: 30px;
+    margin-bottom: 20px;
+    margin-top: 0;
+}
+
+ul ul {
+    font-size: 30px;  
+}
+  }
```
Comment (ericaglimsholt) [inline style.css]: This part needs some work in the indention

File: style.css [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    align-self: center;
+}
+
+p {
+    font-size: 30px;
+    margin-bottom: 20px;
+    margin-top: 0;
+}
+
+ul ul {
```
Comment (ericaglimsholt) [inline style.css]: `ul ul`?

**Pull request closed without merging**